geospaas_harvesting/harvest.yml

```diff
@@ -1,36 +1,63 @@
-      - 'https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/amsr2_conc/catalog.html'
-  podaac:
-    class: 'PODAACHarvester'
-    max_fetcher_threads: 30
-    max_db_threads: 1
-    urls:
-      - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/contents.html'
-      - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_N20/'
-      - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/'
-      - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/MODIS_A/'
-  copernicus_sentinel:
-    class: 'CopernicusSentinelHarvester'
-    max_fetcher_threads: 20
-    max_db_threads: 1
-    url: 'https://scihub.copernicus.eu/apihub/search'
-    search_terms:
-      - 'platformname:Sentinel-1 AND NOT L0'
-      - 'platformname:Sentinel-2 AND NOT L0'
-      - 'platformname:Sentinel-3 AND NOT L0'
-    username: 'topvoys'
-    # Environment variable name
-    password: 'COPERNICUS_OPEN_HUB_PASSWORD'
+  endless: False
+  dump_on_interruption: True
+  poll_interval: 600
+  harvesters:
+    OSISAF:
+      class: 'OSISAFHarvester'
+      max_fetcher_threads: 30
+      # We exclude "EASE-Grid map projections" and "southern hemispheres" from harvesting process
+      excludes: ['ease', '_sh_polstere',]
+      max_db_threads: 1
+      urls:
+        - 'https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/conc/catalog.html'
+        - 'https://thredds.met.no/thredds/catalog/osisaf/met.no/ice/amsr2_conc/catalog.html'
+    podaac:
+      class: 'PODAACHarvester'
+      max_fetcher_threads: 30
+      max_db_threads: 1
+      urls:
+        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/NAVO/v1/2014/005/contents.html'
+        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_N20/'
+        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/VIIRS_NPP/'
+        - 'https://opendap.jpl.nasa.gov/opendap/allData/ghrsst/data/GDS2/L2P/MODIS_A/'
+    copernicus_sentinel:
+      class: 'CopernicusSentinelHarvester'
+      max_fetcher_threads: 20
+      max_db_threads: 1
+      url: 'https://scihub.copernicus.eu/apihub/search'
+      search_terms:
+        - 'platformname:Sentinel-1 AND NOT L0'
+        - 'platformname:Sentinel-2 AND NOT L0'
+        - 'platformname:Sentinel-3 AND NOT L0'
+      username: 'topvoys'
+      # Environment variable name
+      password: 'COPERNICUS_OPEN_HUB_PASSWORD'
+    FTPremss:
+      class: 'FTPHarvester'
+      max_fetcher_threads: 30 # 30
+      max_db_threads: 1
+      username:  arash.azamifard@nersc.no
+      password: 'REMSS_PASSWORD'
+      fileformat: '.gz'
+      urls:
+        - 'ftp://ftp.remss.com/gmi/bmaps_v08.2/y2014/m06/'
+    FTPjaxa:
+      class: 'FTPHarvester'
+      max_fetcher_threads: 30 # 30
+      max_db_threads: 1
+      username: ara12
+      password: JAXA_PASSWORD
```

Comment:
The same regarding open passwords. Now anyone can come and open JAXA without registration. It is not what they want.

Reply:
https://github.com/nansencenter/django-geo-spaas-harvesting/pull/22#discussion_r485527695